myapp/spec/system/task_spec.rb

```diff
@@ -0,0 +1,72 @@
+require 'rails_helper'
+
+RSpec.describe 'Task', type: :system do
+  before do
+    @user = create(:user)
+    @task1 = create(:task, title: 'Test Task 1', status: 0)
+    @task2 = create(:task, title: 'Test Task 2', status: 1)
+    @task3 = create(:task, title: 'Test Task 3', status: 2)
+  end
+
+  describe 'Task list page' do
+    it 'displays a list of tasks' do
+      visit task_index_path
+
+      # check that the tasks are in the list
+      expect(page).to have_content('Test Task 1')
+      expect(page).to have_content('Test Task 2')
+      expect(page).to have_content('Test Task 3')
```

Comment:
then here we can check like below

```suggestion
      expect(page).to have_content(task1.title)
      expect(page).to have_content(task2.title)
      expect(page).to have_content(task3.title)
```

Reply:
great! will do.

Reply:
modified